### DISCUSSION TITLE
Welcome to DEPSTAR-CE Discussions!

### DISCUSSION THREAD

**Discussion by prince-jagani** [prince-jagani/DEPSTAR-CE#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NEELDUDHAT** [comment]
Thank you so much to joining me to in this discussion 😊🙂
Give my regards to hemanshu and vikas
